--- vagrant-2.1.2/vagrant.gemspec~	2018-07-15 10:36:09.000000000 +0200
+++ vagrant-2.1.2/vagrant.gemspec	2018-07-15 10:41:18.155322829 +0200
@@ -22,7 +22,7 @@
   s.add_dependency "listen", "~> 2.7.11"
   s.add_dependency "hashicorp-checkpoint", "~> 0.1.5"
   s.add_dependency "log4r", "~> 1.1.9", "< 1.1.11"
-  s.add_dependency "net-ssh", "~> 4.2.0"
+  s.add_dependency "net-ssh", ">= 4.2.0"
   s.add_dependency "net-sftp", "~> 2.1"
   s.add_dependency "net-scp", "~> 1.2.0"
   s.add_dependency "rest-client", ">= 1.6.0", "< 3.0"
