--- vagrant-1.7.3/vagrant.gemspec	2015-08-19 15:49:13.811643302 +0300
+++ vagrant-1.7.4/vagrant.gemspec	2015-08-19 16:56:50.654693192 +0300
@@ -25,12 +25,7 @@
   s.add_dependency "net-ssh", "~> 4.2.0"
   s.add_dependency "net-sftp", "~> 2.1"
   s.add_dependency "net-scp", "~> 1.2.0"
-  s.add_dependency "rb-kqueue", "~> 0.2.0"
   s.add_dependency "rest-client", ">= 1.6.0", "< 3.0"
-  s.add_dependency "wdm", "~> 0.1.0"
-  s.add_dependency "winrm", "~> 2.1"
-  s.add_dependency "winrm-fs", "~> 1.0"
-  s.add_dependency "winrm-elevated", "~> 1.1"
 
   # NOTE: The ruby_dep gem is an implicit dependency from the listen gem. Later versions
   # of the ruby_dep gem impose an aggressive constraint on the required ruby version (>= 2.2.5).
