--- vagrant-1.5.0/bin/vagrant~	2014-03-10 07:50:03.000000000 +0200
+++ vagrant-1.5.0/bin/vagrant	2014-03-13 07:17:05.591081469 +0200
@@ -137,11 +137,6 @@
   logger.debug("Creating Vagrant environment")
   env = Vagrant::Environment.new(opts)
 
-  if !Vagrant.in_installer?
-    # If we're not in the installer, warn.
-    env.ui.warn(I18n.t("vagrant.general.not_in_installer") + "\n", prefix: false)
-  end
-
   begin
     # Execute the CLI interface, and exit with the proper error code
     exit_status = env.cli(argv)
