--- vagrant-1.5.0/lib/vagrant/shared_helpers.rb~	2014-03-10 07:50:03.000000000 +0200
+++ vagrant-1.5.0/lib/vagrant/shared_helpers.rb	2014-03-13 07:15:33.997139002 +0200
@@ -45,6 +45,7 @@
   #
   # @return [Pathname]
   def self.source_root
+    @source_root ||= Pathname.new('/usr/share/vagrant')
     @source_root ||= Pathname.new(File.expand_path('../../../', __FILE__))
   end
 
--- ./bin/vagrant~	2014-03-13 07:24:35.000000000 +0200
+++ ./bin/vagrant	2014-03-13 09:48:05.406002464 +0200
@@ -42,7 +42,7 @@
   ruby_path = File.join(RbConfig::CONFIG["bindir"], RbConfig::CONFIG["ruby_install_name"])
   Kernel.exec(
     ruby_path,
-    File.expand_path("../../lib/vagrant/pre-rubygems.rb", __FILE__),
+    File.expand_path("/usr/share/gems/gems/vagrant-1.7.1/lib/vagrant/pre-rubygems.rb", __FILE__),
     *ARGV)
   raise "Fatal error: this line should never be reached"
 end
