--- vagrant-1.2.2/./templates/commands/init/Vagrantfile.erb~	2013-04-23 23:05:18.000000000 +0300
+++ vagrant-1.2.2/./templates/commands/init/Vagrantfile.erb	2013-06-07 18:57:30.102493291 +0300
@@ -27,6 +27,10 @@
   # your network.
   # config.vm.network :public_network
 
+  # If true, then any SSH connections made will enable agent forwarding.
+  # Default value: false
+  # config.ssh.forward_agent = true
+
   # Share an additional folder to the guest VM. The first argument is
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
